examples/alphaTestExamples/MultipleApps.md

```diff
@@ -0,0 +1,142 @@
+[kind]: https://github.com/kubernetes-sigs/kind
+
+# Demo: Multiple Apps
```

Comment:
Multiple apps or multiple services?

We should probably stick to one or the other term to avoid confusion.
Both terms are overloaded, but maybe we should stick with `services` here.